includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1162,54 +1156,6 @@ public function generate_checkbox_html( $key, $data ) {
 		return parent::generate_checkbox_html( $key, $data );
 	}
 
-	/**
-	 * Outputs the container for account status information.
-	 *
-	 * @return string Container markup or empty if the account is not connected.
-	 */
-	public function generate_account_status_html() {
-		if ( ! $this->is_connected() ) {
-			return '';
-		}
-
-		ob_start();
-		?>
-		<tr valign="top">
-			<th scope="row">
-				<?php echo esc_html( __( 'Account status', 'woocommerce-payments' ) ); ?>
-			</th>
-			<td>
-				<div id="wcpay-account-status-container"></div>
-			</td>
-		</tr>
-		<?php
-		return ob_get_clean();
-	}
-
-	/**
-	 * Generates markup for the fees information section.
-	 *
-	 * @return string Markup or empty if the account is not connected.
-	 */
-	public function generate_account_fees_html() {
-		if ( ! $this->is_connected() || empty( $this->account->get_fees() ) ) {
```

Comment:
Any idea why the fees object might ever be empty? Do we need to handle it in JS as well?

Reply:
Not sure about this, ideally it never should be.
Any idea @Automattic/gamma ?

Reply:
Currently, the fees will always be provided.

The original code was both server and client changes, so likely it's just an assertive construct for the case the server part is getting reverted.